tests/local_only/uoregon_v2_validation.py

```
@@ -0,0 +1,152 @@
+            v2.da_ipv6_count
+        FROM v1.ip_stats v1
+        JOIN v2.ip_stats_v2 v2
+          ON v2.source_id = v1.router
+         AND v2.granularity = v1.granularity
+         AND v2.bucket_start = v1.bucket_start
+        WHERE v1.router = ?
+          AND v1.bucket_start >= ?
+          AND v1.bucket_start < ?
+          AND v1.granularity IN ({granularity_placeholders()})
+        ORDER BY v1.granularity, v1.bucket_start
+        """,
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)